src/index.jsx

```diff
@@ -9,48 +9,28 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { Route, Switch } from 'react-router-dom';
 
-import Footer, { messages as footerMessages } from '@edx/frontend-component-footer';
+import { messages as footerMessages } from '@edx/frontend-component-footer';
 
 import appMessages from './i18n';
-import { CoursePageResources } from './course-page-resources';
-import ProctoredExamSettings from './proctored-exam-settings/ProctoredExamSettings';
-import StudioHeader from './studio-header/Header';
 
 import initializeStore from './store';
 import './index.scss';
 import './assets/favicon.ico';
+import CourseAuthoringRoutes from './CourseAuthoringRoutes';
+import LegacyProctoringRoute from './proctored-exam-settings/LegacyProctoringRoute';
 
 subscribe(APP_READY, () => {
   ReactDOM.render(
     <AppProvider store={initializeStore()}>
       <Switch>
-        <Route
-          path="/proctored-exam-settings/:course_id"
-          exact
-          render={({ match }) => {
-            const courseId = decodeURIComponent(match.params.course_id);
-            return (
-              <>
-                <StudioHeader courseId={courseId} />
-                <ProctoredExamSettings courseId={courseId} />
-              </>
-            );
-          }}
-        />
-        <Route
-          path="/course-pages/:course_id"
-          render={({ match }) => {
-            const courseId = decodeURIComponent(match.params.course_id);
-            return (
-              <>
-                <StudioHeader courseId={courseId} />
-                <CoursePageResources courseId={courseId} />
-              </>
-            );
-          }}
-        />
+        <Route exact path="/proctored-exam-settings/:courseId">
+          {/* See component for details on what this is */}
+          <LegacyProctoringRoute />
+        </Route>
+        <Route path="/course/:courseId">
+          <CourseAuthoringRoutes />
+        </Route>
```

Comment:
Side note: when you make the corresponding edx-platform PR to update legacy Studio to the new URL scheme, can you make sure that the encoding of the course key in the URL is fixed?  It's been driving me crazy!

Example:

Currently, if I click "Proctored Exam Settings" from
```
https://studio.stage.edx.org/course/course-v1:edX+BD-14+2020
```
I get sent to:
```
https://course-authoring.stage.edx.org/proctored-exam-settings/course-v1%3AedX%2BBD-14%2B2020
```

I'd hope that the new link is:
```
https://course-authoring.stage.edx.org/course-v1:edX+BD-14+2020/proctored-exam-settings
```
instead of 
```
https://course-authoring.stage.edx.org/course-v1%3AedX%2BBD-14%2B2020/proctored-exam-settings
```